src/main/java/com/amazon/opendistroforelasticsearch/sql/antlr/semantic/types/function/ScalarFunction.java

```diff
@@ -43,8 +43,8 @@
     COSH(func(T(NUMBER)).to(T)),
     COT(func(T(NUMBER)).to(T)),
     DATE_FORMAT(
-        func(DATE, STRING).to(STRING),
-        func(DATE, STRING, STRING).to(STRING)
+        func(ESDataType.DATE, STRING).to(STRING),
```

Comment:
What is the reason behind this change?

Reply:
Because we added a scalar function named `DATE` in the same class. This made some confusion to the "DATE" type.